binplace.targets

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" InitialTargets="CheckForBuildTools" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <IsRuntimeAssembly Condition="'$(IsRuntimeAssembly)'=='' AND '$(IsReferenceAssembly)' != 'true'">true</IsRuntimeAssembly>
+    <BinPlaceStuffDependsOn Condition="'$(IsReferenceAssembly)' == 'true'">$(BinPlaceStuffDependsOn);BinPlaceReferenceAssembly</BinPlaceStuffDependsOn>
+    <BinPlaceStuffDependsOn Condition="'$(IsRuntimeAssembly)' == 'true'">$(BinPlaceStuffDependsOn);BinPlaceRuntimeAssembly</BinPlaceStuffDependsOn>
+  </PropertyGroup>
+
+  <Target Name="BinPlaceStuff" DependsOnTargets="$(BinPlaceStuffDependsOn)" AfterTargets="CopyFilesToOutputDirectory">
+    <Message Importance="low" Text="IsRuntimeAssembly: $(IsRuntimeAssembly)" />
+    <Message Importance="low" Text="IsReferenceAssembly: $(IsReferenceAssembly)" />
+  </Target>
+
+  <Target Name="BinPlaceReferenceAssembly" DependsOnTargets="GetTargetingPackDir">
```

Comment:
That sounds good to me. As for having separate targets: it's probably not necessary. I think we may have multiple copies happening in a single build of a project, though. Like I mentioned in our chat today, some of the projects drop the same assembly in two places, like stuff built on top of .NET Standard, etc.